kartothek/io_components/docs.py

```diff
@@ -31,7 +31,8 @@
     "columns": """
     columns : dict of list of string, optional
         A dictionary mapping tables to list of columns. Only the specified
-        columns are loaded for the corresponding table.""",
+        columns are loaded for the corresponding table. If a specfied table is
+        not present in the dataset, a ValueError is raised.""",
```

Comment:
Does this also raise if the specified column does not exist in the dataset? Please add that info as well

Reply:
Added it